llvm/test/SYCLLowerIR/esimd_lower_inline_hint.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: opt -LowerESIMD -S < %s | FileCheck %s
+
+; This test checks that LowerESIMD pass sets the 'alwaysinline'
+; attribute for all the functions called from ESIMD kernels.
```

Comment:
There is no ESIMD specifics in the test, and LowerESIMD will add the attr to any non-kernel function it meets. Could you update the comment please?

Reply:
I think there are still implicit ESIMD-specifics since we intend to run the LowerESIMD pass only on the ESIMD code. I fixed the comment.